### PR TITLE
Release v0.7.0

### PR DESCRIPTION
**Pull request by codysoyland**
## Overview

This adds the changelog and installation instructions for Pilosa v0.7.0.

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,51 @@ All notable changes to this project will be documented in this file.
...
+- Add HTTP API for fields ([#811](https://github.com/pilosa/pilosa/pull/811), [#856](https://github.com/pilosa/pilosa/pull/856))
+- Add HTTP API for delete views ([#785](https://github.com/pilosa/pilosa/pull/785))
+- Modify import endpoint to handle BSI field values ([#840](https://github.com/pilosa/pilosa/pull/840))
+- Add field Range() support to Executor ([#791](https://github.com/pilosa/pilosa/pull/791))
+- Support PQL Range() queries for fields ([#755](https://github.com/pilosa/pilosa/pull/755))
+- Add Sum() and Average() field queries ([#778](https://github.com/pilosa/pilosa/pull/778))
+- Add documentation for BSI ([#861](https://github.com/pilosa/pilosa/pull/861))
+- Add BETWEEN for Range queries ([#847](https://github.com/pilosa/pilosa/pull/847))
+- Add Xor support for PQL ([#789](https://github.com/pilosa/pilosa/pull/789))
+- Enables auto-creating the schema on imports ([#837](https://github.com/pilosa/pilosa/pull/837))
```
Comment (travisturner) [inline CHANGELOG.md]: a couple of these verbs aren't consistent:
Enables
Updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,51 @@ All notable changes to this project will be documented in this file.
...
+- Fix infinite loop in bitmap-to-array conversion ([#779](https://github.com/pilosa/pilosa/pull/779))
+- Fix Countrange bug ([#773](https://github.com/pilosa/pilosa/pull/773))
+
+### Removed
+
+- Remove support for row/column labels ([#839](https://github.com/pilosa/pilosa/pull/839))
+
+### Performance
+
+- Refactor differencerunarray ([#859](https://github.com/pilosa/pilosa/pull/859))
```
Comment (travisturner) [inline CHANGELOG.md]: can we change this to `differenceRunArray`

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,51 @@ All notable changes to this project will be documented in this file.
...
+- Add ability to suprress attributes and bits in Bitmap queries ([#783](https://github.com/pilosa/pilosa/pull/783))
+
+### Fixed
+
+- Fix panic when iterating over an empty run container ([#860](https://github.com/pilosa/pilosa/pull/860))
+- Fix row id zero bug ([#814](https://github.com/pilosa/pilosa/pull/814))
+- Fix cache invalidation bug ([#795](https://github.com/pilosa/pilosa/pull/795))
+- Set container.n in differenceRunRun ([#794](https://github.com/pilosa/pilosa/pull/794))
+- Fix infinite loop in bitmap-to-array conversion ([#779](https://github.com/pilosa/pilosa/pull/779))
+- Fix Countrange bug ([#773](https://github.com/pilosa/pilosa/pull/773))
```
Comment (travisturner) [inline CHANGELOG.md]: change to `CountRange`

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,51 @@ All notable changes to this project will be documented in this file.
...
+- Support PQL Range() queries for fields ([#755](https://github.com/pilosa/pilosa/pull/755))
+- Add Sum() and Average() field queries ([#778](https://github.com/pilosa/pilosa/pull/778))
+- Add documentation for BSI ([#861](https://github.com/pilosa/pilosa/pull/861))
+- Add BETWEEN for Range queries ([#847](https://github.com/pilosa/pilosa/pull/847))
+- Add Xor support for PQL ([#789](https://github.com/pilosa/pilosa/pull/789))
+- Enable auto-creating the schema on imports ([#837](https://github.com/pilosa/pilosa/pull/837))
+- Update client library docs ([#831](https://github.com/pilosa/pilosa/pull/831))
+- Handle SIGTERM signal ([#830](https://github.com/pilosa/pilosa/pull/830))
+- Add cluster config example to docs ([#806](https://github.com/pilosa/pilosa/pull/806))
+- Add ability to suprress attributes and bits in Bitmap queries ([#783](https://github.com/pilosa/pilosa/pull/783))
```
Comment (travisturner) [inline CHANGELOG.md]: suppress

**Comment by yuce** [comment]
@codysoyland `Average()` call was removed in favor of returning sum and count from `Sum()` call.

**Comment by yuce** [comment]
@codysoyland *Add ability to exclude attributes and bits in Bitmap queries* may sound better than *Add ability to suppress attributes and bits in Bitmap queries*.

**Comment by yuce** [comment]
@codysoyland Also, it may be better to rename *Removed* section to *Deprecated* since the actual row/label code is still in place, only the docs are removed.

**Comment by codysoyland** [comment]
Thanks @yuce and @travisturner. Everything is updated now.

**Comment by yuce** [comment]
Thanks @cody.

**Pull request merged**